pages/code-review.md

```diff
@@ -114,12 +149,26 @@ When reviewing code, you should be able to check off each of the following:
     - [ ] Do the changes address the project's needs?
     - [ ] Do the changes respect the project's existing style?
     - [ ] Does the new code avoid reproducing existing functionality?
-    - [ ] Are functions as simple as possible?
+    - [ ] Are functions/classes as simple as possible?
     - [ ] Is the code as efficient as possible?
-    - [ ] Is the usage of each function clear?
+    - [ ] Is the usage of each function/class clear?
     - [ ] Have edge cases been considered and tested for?
-
-Use this bookmarklet code to add this checklist to your Pull Request comments: [Code Review Checklist](https://gist.github.com/cfarm/a4174fe6f775353a3115)
+    - [ ] Does the code represent a logical unit of work?
+    - [ ] Are there any glaring syntax errors that were missed?
+    - [ ] Are language constructs being utilized properly?
+    - [ ] Are there no unecessary premature optimizations?
+    - [ ] Are any frameworks/libraries being used leveraged properly?
+    - [ ] Are the changes clean and would they pass through a linter?
+    - [ ] Are there useful comments/documentation where needed?
+    - [ ] Is the commit message well-formed with succinct details about
+          what the change is intended for?
+
+In adition to this checklist, you should also pull down the actual changes
+and run them manually to put them through their paces.  Does the code
```

Comment:
I understand the concern that not all pull requests will warrant needing to go through the step of running things manually.  Truly dead simple changes should not soak up a lot of time, I agree!

I also understand the concern around prescribing what all teams **must** do as a part of their code reviews.  However, I am looking at the development guide through the lens of helping establish good baseline practices at 18F and leading by example.  In my eyes that translates into performing the necessary due diligence to ensure high quality work is consistently delivered and clearly documenting what that process entails.

In the case of code reviews, I feel that pulling down the changes; making sure that they do merge seamlessly (especially if there are any schema or data migrations involved); and actually using the code, even if just a high level pass, is an important part of the review process and helps immensely in ensuring high quality work.  Speaking from personal experience, I have been bitten far more times than I care to admit by the "did you actually run the code?" question when it comes to signing off or giving the thumbs up on someone else's work only to have something fail in production.

All of that said, if the sticking point is about what any given team *should* do versus what they *ought* to do to as a part of the code review process, what if the language was changed to say "At you or your team's discretion you may also consider pulling down the changes yourself and running through the code locally as a part of the review," or something to that effect?  Would that account for situations where this step is indeed overkill and preserve the autonomy of the teams, but still call out that this is indeed a useful exercise to consider?